src/main/webapp/WEB-INF/jsp/all-section-deals.jsp

```diff
@@ -32,6 +32,7 @@
           <a class="dropdown-item" href="#">Trending</a>
           <a class="dropdown-item" href="#">Votes</a>
           <a class="dropdown-item" href="#">New</a>
+          <a id="sort-dist" class="dropdown-item" href="#" onclick="return getLocation();">Distance</a>
```

Comment:
what's the return for?

Reply:
like if return false then the href wont be loaded